crates/oxc_linter/src/snapshots/aria_role.snap

```diff
@@ -0,0 +1,117 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 144
+expression: aria_role
+---
+  ⚠ eslint-plugin-jsx-a11y(aria-role):Elements with ARIA roles must use a valid, non-abstract ARIA role.
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='foobar' />
+   ·      ─────────────
+   ╰────
+  help: Set a valid, non-abstract ARIA role for element with ARIA
+
+  ⚠ eslint-plugin-jsx-a11y(aria-role):Elements with ARIA roles must use a valid, non-abstract ARIA role.
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='datepicker'></div>
+   ·      ─────────────────
+   ╰────
+  help: Set a valid, non-abstract ARIA role for element with ARIA
+
+  ⚠ eslint-plugin-jsx-a11y(aria-role):Elements with ARIA roles must use a valid, non-abstract ARIA role.
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='range'></div>
+   ·      ────────────
+   ╰────
+  help: Set a valid, non-abstract ARIA role for element with ARIA
+
+  ⚠ eslint-plugin-jsx-a11y(aria-role):Elements with ARIA roles must use a valid, non-abstract ARIA role.
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='Button'></div>
+   ·      ─────────────
+   ╰────
+  help: Set a valid, non-abstract ARIA role for element with ARIA
+
+  × Unterminated string
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='></div>
+   ·           ────────
+   ╰────
+
+  ⚠ eslint-plugin-jsx-a11y(aria-role):Elements with ARIA roles must use a valid, non-abstract ARIA role.
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='tabpanel row foobar'></div>
+   ·      ──────────────────────────
+   ╰────
+  help: Set a valid, non-abstract ARIA role for element with ARIA
+
+  ⚠ eslint-plugin-jsx-a11y(aria-role):Elements with ARIA roles must use a valid, non-abstract ARIA role.
+   ╭─[aria_role.tsx:1:1]
+ 1 │ <div role='tabpanel row range'></div>
+   ·      ─────────────────────────
```

Comment:
for these cases, it would be good if we could point to the exact role that's incorrect.

Alternatively, we could say which role(s) are incorrect in the error/help message